AccessHandle.md

```diff
@@ -0,0 +1,247 @@
+# Adding AccessHandles to files
+
+## Authors:
+
+* Emanuel Krivoy (fivedots@chromium.org)
+* Richard Stotz (rstz@chromium.org)
+
+## Participate
+
+* [Issue tracker](https://github.com/WICG/file-system-access/issues)
+
+## Table of Contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Introduction](#introduction)
+- [Goals and Use Cases](#goals-and-use-cases)
+- [Non-goals](#non-goals)
+- [Proposed API](#proposed-api)
+  - [New data access surface](#new-data-access-surface)
+  - [Locking semantics](#locking-semantics)
+- [Open Questions](#open-questions)
+  - [Naming](#naming)
+  - [Assurances on non-awaited consistency](#assurances-on-non-awaited-consistency)
+- [Appendix](#appendix)
+  - [AccessHandle IDL](#accesshandle-idl)
+- [Stakeholder Feedback / Opposition](#stakeholder-feedback--opposition)
+- [References & acknowledgements](#references--acknowledgements)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Introduction
+
+This is a proposal to support
+[Storage Foundation API's](https://github.com/WICG/storage-foundation-api-explainer)
```

Comment:
Consider reducing the focus on Storage Foundation API to improve readability

Reply:
I've rewritten a big part of the doc, could you PTAL?